easybuild/easyconfigs/s/SPIDER/SPIDER-22.10-intel-2015a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'SPIDER'
+version = '22.10'
+
+homepage = 'http://spider.wadsworth.org/spider_doc/spider/docs/spider.html'
+description = """SPIDER (System for Processing Image Data from Electron microscopy and Related fields) is
+ an image processing system for electron microscopy."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'static': True}
+
+source_urls = ['http://spider.wadsworth.org/spider_doc/spider/download/']
+sources = ['spiderweb.%(version)s.tar.gz']
+
+patches = ['SPIDER-%(version)s_fix-syntax-ifort.patch']
+
+dependencies = [('FFTW', '3.3.4')]
+
+start_dir = 'spider/src'
+
+buildininstalldir = True
+
+parallel = 1
+
+skipsteps = ['configure', 'install']
+
+prebuildopts = "rm ../bin/spider* && rm ../src/spider*.[ao] && "
+buildopts = '-f Makefile_linux COMP="$F90" EXE=spider FFTWLIBDIR=$EBROOTFFTW/lib '
+buildopts += 'FFLAGS="$F90FLAGS -WB -DSP_LIBFFTW3 -DSP_LINUX -DSP_IFC -fpp -c" LF="$F90FLAGS"'
+#buildopts += 'FFLAGS="$F90FLAGS -DHAS_IEEE -DSP_LIBFFTW3 -DSP_LINUX -DSP_IFC -DSP_MP -c" LF="$F90FLAGS"'
```

Comment:
ah, this is leftover crap, thanks;

I need to look into the `-mp` and `-mpi` variants too, forgot about that...